index.html

```diff
@@ -511,29 +759,30 @@ <h2>
       </h2>
       <p>
         A <dfn>default unlock gesture</dfn> must always be available that will
-        exit pointer lock.
+        [=exit pointer lock=] with the [=user agent=]'s [=pointer-lock
+        target=].
       </p>
       <aside class="note">
         <p>
-          The ESC key is the recommended <a>default unlock gesture</a> for
-          <a>user agents</a> with keyboard input. It is recommended that the
-          unlock gesture also match any used to exit fullscreen [[FULLSCREEN]].
+          The ESC key is the recommended [=default unlock gesture=] for [=user
+          agents=] with keyboard input. It is recommended that the unlock
+          gesture also match any used to exit fullscreen [[FULLSCREEN]].
```

Comment:
This is again not idea... fully exiting full screen should release the lock also! 

We should tie these APIs together really otherwise the user would have to hit "ESC" multiple times.

Reply:
Done.